scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +64,21 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val, metric_type):
+        """
+        :param val: string representing metric value
+        :param metric_type: string or numeric
+        :return: value converted to float if metric_type isn't string
+        """
+        if metric_type == 'string':
+            return val
+
+        try:
+            return float(val)
```

Comment:
Hmm, have you tested this against Zabbix?

Zabbix has int and float as 2 separate types. So I'm worried that all of our int based metrics will fail if this float conversion changes the value from 5 to 5.0 when parsed to json.